README.md

```diff
@@ -1,8 +1,307 @@
+
+#### Spawn subnet actor
+To run a subnet the first thing is to configure and create the subnet actor that will govern the subnet's operation:
+```bash
+./bin/ipc_agent create-subnet -p <parent-id> -n <subnet-name> --min-validator-stake 1 --min-validators <num-validators> --finality-threshold <number-epochs> --check-period <epochs-between-checks>
+
+# Sample command execution
+./bin/ipc_agent create-subnet -p /root -n test --min-validator-stake 1 \
+--min-validators 0 --finality-threshold 10 --check-period 10
+
+[2023-03-21T09:32:58Z INFO  ipc_agent::cli::commands::manager::create] created subnet actor with id: /root/t01002
+```
+This command deploys a subnet actor for a new subnet from the `root`, with a human-readable name `test`, that requires at least `1` validator to join the subnet to be able to mine new blocks, and with a checkpointing period to the parent of `10` blocks. We can see that the output of this command is the ID of the new subnet.
+
+#### Exporting wallet keys from subnet
+In order to run a validator in a subnet, we'll need a set of keys to handle that validator. To export the validator key from a wallet that may live in some other subnet into a file (like the wallet address we are using in the rootnet), we can use the following Lotus command:
+```bash
+eudico wallet export --lotus-json <address-to-export> > <output file>
+
+# Sample execution
+eudico wallet export --lotus-json t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq > ~/.ipc-agent/wallet.key
+```
+If your daemon is running on a docker container, you can get the container id (provided also in the output of the infra scripts), and run the following command above inside a container outputting the exported private key into a file locally:
+```bash
+$ docker exec -it <container-id> eudico wallet export --lotus-json <adress-to-export> > ~/.ipc-agent/wallet.key
+
+# Sample execution
+$ docker exec -it 84711d67cf162e30747c4525d69728c4dea8c6b4b35cd89f6d0947fee14bf908 eudico wallet export --lotus-json t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq > ~/.ipc-agent/wallet.key
+```
+
+Let's illustrate the flow by creating a new wallet in our recently deployed root and exporting the keys.
+```bash
+# Create the new wallet
+$ ./bin/ipc_agent wallet-new --key-type=secp256k1 --subnet=/root
+[2023-03-29T09:32:52Z INFO  ipc_agent::cli::commands::manager::wallet] created new wallet with address WalletNewResponse { address: "t17rnww5qirr2fh5uiqy6fyi6ix7otwjzgu6pgpey" } in subnet "/root"
+
+# Export the created wallet into ipc-agent
+$ docker exec -it <subnet-container-id> eudico wallet export --lotus-json <filecoin-addr> > <output_directory>
+
+# Sample execution for the address created above
+$ docker exec -it 84711d67cf162e30747c4525d69728c4dea8c6b4b35cd89f6d0947fee14bf908 eudico wallet export --lotus-json t17rnww5qirr2fh5uiqy6fyi6ix7otwjzgu6pgpey > ~/.ipc-agent/wallet.key
+```
```

Comment:
Is there a reason to do this here? We've show a new wallet being created above, and how to export right after this. The export above is the one we'll use in the next step. This seems like a bit of an unnecessary detour.